html/index.html

```diff
@@ -1,11 +1,71 @@
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>MALNUTRITION</title>
+        <link rel="stylesheet" href="../css/style.css">
+    </head>
+    <body>
+        
+        <div class="container">
```

Comment:
Use the h1 tag to create a main heading for the webpage. 
I can't seem to find this implemented here